mibidata/tests/test_mibi_image.py

```diff
@@ -127,6 +127,20 @@ def test_check_fov_id(self):
         with self.assertRaises(ValueError):
             image.fov_id = None
 
+    def test_check_fov_id_without_folder(self):
+        metadata = METADATA.copy()
+        metadata.pop('folder')
+        image = mi.MibiImage(TEST_DATA, TUPLE_LABELS, **metadata)
+        self.assertEqual(image.fov_id, 'Point1');
+        self.assertEqual(image.folder, None); # We allow folder to not be provided in the constructor kwargs, it is set to None in this case.
```

Comment:
@rfinck - if you alter constructor behavior (i.e. fix that bug) this line will likely fail. Just pointing that out.

Reply:
Thanks @rileysparsons -- I reverted these tests to the original ones you added which correctly caught this issue, and updated the setters to prevent it. Note that now if you try `image.folder = None` it will leave the folder as-is rather than break consistency with fov_id, and I think that is better behavior.